src/utils/create_component_file/index.ts

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+const path = require('path');
+import componentTemplate from './templates/template_component';
+import testTemplate from './templates/template_test';
+
+// Process arguments
+const componentName = process.argv[2];
+const folderPath = process.argv[3] || '.';
+
+// Check if component name is provided
+if (!componentName) {
+  console.log('You must provide a component name.');
+  process.exit(1);
+}
+
+// Determine the import path
+const importPath = `components/${
+  folderPath === '.' ? '' : folderPath + '/'
+}${componentName}`;
+
+// Ensure the component and tests directories exist
+const projectRoot = process.cwd();
+ensureDir(path.join(projectRoot, `src/components/${folderPath}`));
```

Comment:
Test should create own temporary directory which store dynamically generated file structure e.g. `/tmp/tmpfqq9piuk/src/components/${folderPath}` and delete automatically after test finish.

Reply:
I understand. However, is it not also valuable to test that files are being created in the correct directory within the file structure?

Reply:
Yes it is. For testing purpose you change root dir only (instead of dir ride-to-work-by-bike-frontend/src/components you can use e.g. /tmp/tmpfqq9piuk/src/components/), but file structure is the same.

Create custom temp dir with Node.js

https://blog.mastykarz.nl/create-temp-directory-app-node-js/

Reply:
But yes , is one of another option, if you use real root dir, but is important to preserved `watchForFileChanges: true`.

Reply:
I ended up running the test with `watchForFileChanges: true` without additional changes. It seems that the rerunning of the script in the earlier version of the test was caused by something else.